Plugin.php

```diff
@@ -105,9 +105,17 @@ public function registerCumulusAnnouncerTypes()
     {
         return [
             \Initbiz\CumulusCore\AnnouncerTypes\UserRegisterAnnouncerType::class,
+            \Initbiz\CumulusCore\AnnouncerTypes\OnboardingUsersAnnouncerType::class,
         ];
     }
 
+    public function registerSchedule($schedule)
+    {
+        $schedule->call(function () {
+            Event::fire('initbiz.cumuluscore.sendOnboardingMessages');
+        })->dailyAt("10:00");
```

Comment:
The time should be in the config

Reply:
It definitely should. All schedules are hardcoded for now though. As per our chat, I'll leave `10:00` and if we ever need it customised we'll handle that. For the time being, it's fine to have it hardcoded.